widgetastic_manageiq/__init__.py

```diff
@@ -5205,3 +5205,63 @@ def fill(self, value):
 
     def read(self):
         return self.is_set
+
+
+class MultiBoxOrderedSelect(MultiBoxSelect):
+    move_into_button = Button(title=Parameter("@move_into"))
+    move_from_button = Button(title=Parameter("@move_from"))
+    move_up_button = Button(title=Parameter("@move_up"))
+    move_down_button = Button(title=Parameter("@move_down"))
+
+    def __init__(
+        self,
+        parent,
+        available_items,
+        chosen_items,
```

Comment:
We are losing the default ID strings that are defined in `MultiBoxSelect` and used for the Parametrized locators. since there are no defaults here, and these values are passed into the parent constructor.